src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -24,9 +24,28 @@ class NewFurniture extends React.Component {
       this.setState({ activeCategory: newCategory, activePageStyle: styles.fadeIn });
     }, 1000);
   }
+  setChunk() {
+    let chunk;
+    if (this.props.mode === 'desktop') {
+      chunk = 8;
+    } else if (this.props.mode === 'tablet') {
+      chunk = 2;
+    } else if (this.props.mode === 'mobile') {
+      chunk = 1;
+    } else {
+      if (window.innerWidth < 540) {
```

Comment:
Po co to jest skoro sprawdzamy po `mode`? To nie jest to samo?

Reply:
Chodziło mi o uwzględnienie przypadku zaraz po uruchomieniu strony przed resize (jak Maciek sugerował), przeniosłam to teraz do modeRedux jako default, reszta rzeczy poprawiona :) @kamilwilk95